packages/utilities/__tests__/diff/WebDiffGenerator.test.ts

```diff
@@ -0,0 +1,37 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import * as  fs from 'fs';
+import * as path from 'path';
+
+import WebDiffGenerator from "../../src/diff/WebDiffGenerator";
+import { ImperativeConfig } from "../../src/ImperativeConfig";
+import {IO} from '../../../io/';
+
+describe("WebDiffGenerator", () => {
+    // setting up fake cli home and web diff dir for testing
+    const cliHome: string = "packages/__tests__/fakeCliHome";
```

Comment:
Instead of checking the "fakeCliHome" directory into the Git repository, should we create the directory temporarily in a `beforeAll` block before the tests run, and remove it afterwards in an `afterAll` block? This would lessen the amount of work needed to maintain this test in case the contents of the CLI home dir change in the future.